packages/commons/src/test/utils.ts

```diff
@@ -0,0 +1,40 @@
+import { GenericContainer } from "testcontainers";
+import { PostgreSqlContainer } from "@testcontainers/postgresql";
+
+export const getMongodbContainer = ({
+  dbName,
+  username,
+  password,
+}: {
+  username: string;
+  password: string;
+  dbName: string;
+}): GenericContainer =>
+  new GenericContainer("mongo:6.0.7")
```

Comment:
we are using [DocumentDB on AWS](https://docs.aws.amazon.com/en_en/documentdb/latest/developerguide/mongo-apis.html) which supports until `Mongo 5.0.x` , I suggest to align the version to avoid unexpected behaviours

Reply:
the DocumentDB version we are using (partially) supports MongoDB version `4.0.x`

Reply:
Thanks, updated to `4.0.0`